chesster.js

```diff
@@ -161,6 +152,17 @@ function update_channels(bot){
     });
 
 }
+function refreshCurrentRoundSchedules() {
+    var _45_45 = league.getLeague("45+45", config);
+    _45_45.getCurrentRoundSchedule(function(err, pairings) {
```

Comment:
I like where this is going, but since the leagues will likely update more than round schedules, I think we should simply have a `refresh` function

```
var _45_45 = leage.getLeague("45+45", config);
_45_45.refresh();
```

where `refresh` calls `getCurrentRoundSchedule`, `getMods`, `getRosters`, etc